### PR TITLE
Log location & file name Update

### PR DESCRIPTION
**Pull request by SudarsanPradeepKumar**
The log location & filename for the AEM CS logging configuration is incorrect.

### CODE REVIEW

File: help/implementing/developing/introduction/development-guidelines.md [hunk 1]
```diff
@@ -100,11 +100,20 @@ To change the log levels for Cloud environments, the Sling Logging OSGI configur
...
 
 The default log level is INFO, that is, DEBUG messages are not logged.
-To activate DEBUG log level, set the
+To activate DEBUG log level, update the org.apache.sling.commons.log.level property to debug
 
-``` /libs/sling/config/org.apache.sling.commons.log.LogManager/org.apache.sling.commons.log.level ```
+The following is an example of the logging configurations
 
-property to debug. Do not leave the log at the DEBUG log level longer than necessary, as it generates a lot of logs.
+/apps/my-app/config/org.apache.sling.commons.log.LogManager.factory.config.cfg.json
```
Comment (davidjgonzalez) [inline help/implementing/developing/introduction/development-guidelines.md:107]:
```suggestion
`/apps/example/config/org.apache.sling.commons.log.LogManager.factory.config~example.cfg.json`
```

File: help/implementing/developing/introduction/development-guidelines.md [hunk 2]
```diff
@@ -100,11 +100,20 @@ To change the log levels for Cloud environments, the Sling Logging OSGI configur
...
+
+```
+{
+    "org.apache.sling.commons.log.names": ["com.example"],
+    "org.apache.sling.commons.log.level": "debug"
+}
```
Comment (davidjgonzalez) [inline help/implementing/developing/introduction/development-guidelines.md:113]: {
  "org.apache.sling.commons.log.names": [
    "com.example"
  ],
  "org.apache.sling.commons.log.level": "DEBUG",
  "org.apache.sling.commons.log.file": "logs/error.log",
  "org.apache.sling.commons.log.additiv": "false"
}

Comment (davidjgonzalez) [reply help/implementing/developing/introduction/development-guidelines.md:113]: This is the Log config generated by latest AEM Archetype

File: help/implementing/developing/introduction/development-guidelines.md [hunk 3]
```diff
@@ -100,11 +100,20 @@ To change the log levels for Cloud environments, the Sling Logging OSGI configur
...
+```
+
+Do not leave the log at the DEBUG log level longer than necessary, as it generates a lot of logs.
```
Comment (davidjgonzalez) [inline help/implementing/developing/introduction/development-guidelines.md:116]:
```suggestion
Do not leave the log at the DEBUG log level longer than necessary, as it generates a lot of logs. 

Discrete log levels can be set for the different AEM environments using runmode-based OSGi configuration targetting. For example:

| Environment | OSGi configuration location by run mode | `org.apache.sling.commons.log.level` property value |
| - | - | - |
|  Development | /apps/example/config/org.apache.sling.commons.log.LogManager.factory.config~example.cfg.json | DEBUG |
|  Stage | /apps/example/config.stage/org.apache.sling.commons.log.LogManager.factory.config~example.cfg.json | WARN |
|  Production | /apps/example/config.prod/org.apache.sling.commons.log.LogManager.factory.config~example.cfg.json | ERROR |
```

**Review by davidjgonzalez — COMMENTED** [review]
WDYT?

**Comment by bohnertchris** [comment]
Thanks for the suggestion, @SudarsanPradeepKumar and for the pointers, @davidjgonzalez. Looks good and I am moving this to our corporate git as PR 2811

**Comment by bohnertchris** [comment]
This was fixed with commit e7ce29c

**Pull request closed without merging**